### ISSUE TITLE
Dev: WAVE Page Audit: Forgot Password

### ISSUE THREAD

**Issue by JRParryY** [hackforla/tdm-calculator#2701]
### Overview
We need to run the WAVE evaluation browser extension on [Forgot Password] to audit the page for WCAG2.2 accessibility compliance. 

#### Details
We will need to audit all user interactions on the page so that all possible states and user flows are compliant.
- Page URL: https://tdm-dev.azurewebsites.net/forgotpassword
  - Requires sign in: [false]
    - User Profile Required: [None]
  - Can access page directly from URL: [true]

### Action Items

- [ ] Review wiki page: Accessibility, **How to run the WAVE browser extension (Resource 1.01.02)**
- [ ] Install the WAVE browser extension
  - (suggested) pin to your browser's toolbar for quick access
- [ ] Open TDM WAVE Accessibility Audit Spreadsheet, **1. Page Log (Resource 1.02.01)**
  - [ ] Choose the row that has your issue number and populate the following fields...
    - [ ] Populate `Auditor` cell with your GitHub handle
    - [ ] Populate `Begin Date` with today's date (YYYY-MM-DD)
  - [ ] Log in as the user indicated under Details and navigate to the page you will audit
    - [ ] Run the WAVE extension with no elements focused, or clicked
      - [ ] In TDM WAVE Accessibility Audit Spreadsheet, **2. Audit Log (Resource 1.02.02)**, create a row for every Error, Contrast Error, and Alert on the Details pane
        - [ ] populate `Auditor` with your GitHub handle
        - [ ] populate `Page` with your selected page from **1. Page Log**
        - [ ] populate `Error Title` with the dropdown that matches the subtitle on the Details pane (see **Resource 2.01.02**)
        - [ ] double check that the `Category` field now matches the title on the Details pane (see **Resource 2.01.01**). If there's not a match, leave a comment with the actual title 
        - [ ] populate the `DOM snippet` by pasting in the highlighted HTML snippet for the Code panel
        - [ ] leave `Steps to Reproduce` empty

- Form-Elements
    - [ ] For all form elements (e.g. text input, password fields, checkboxes), attempt to trigger validation error messages (e.g. missing input, password complexity, incorrect username or password)
      - [ ] In **2. Audit Log**, create and populate row for every Error, Contrast Error, and Alert on the Details pane
        - [ ] populate `Auditor` with your GitHub handle
        - [ ] populate `Page` with your selected page from **1. Page Log**
        - [ ] populate `Error Title` with the dropdown that matches the subtitle on the Details pane (see **Resource 2.01.02**)
        - [ ] double check that the `Category` field now matches the title on the Details pane (see **Resource 2.01.01**). If there's not a match, leave a comment with the actual title
        - [ ] populate the `DOM snippet` by pasting in the highlighted HTML snippet for the Code panel
        - [ ] in a new comment in this issue, write out all steps required to trigger the validation message
          - [ ] populate `Steps to Reproduce` with the link to this comment

### Resources/Instructions

- 1.01 TDM Wiki Pages
  - 1.01.01 [Accessibility Audits](https://github.com/hackforla/tdm-calculator/wiki/Accessibility-Audits)
    - 1.01.02 [How to run the WAVE browser extension](https://github.com/hackforla/tdm-calculator/wiki/Accessibility-Audits#tdm-wave-audits)
- 1.02 _TDM WAVE Accessibility Audit Spreadsheet_
  - 1.02.01 [1. Page Log](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=669073341#gid=669073341)
  - 1.02.02 [2. Audit Log](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0)
  - 1.02.03 [3. Solved Index](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=406857693#gid=406857693)
  - 1.02.04 [4. Unsolved Index](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=1793119266#gid=1793119266)
  - 1.02.05 [5. WAVE Error Index](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=1697403119#gid=1697403119)

#### Audit Instructions

<details><summary>2.01 WAVE Details pane (title and subtitle)</summary>
<p>

> **2.01.01 Title**
> <img height="640" alt="Image" src="https://github.com/user-attachments/assets/954fbfcf-bcff-4ed1-b85d-ed22ee2d460e" />
>
> **2.01.02 Subtitle**
> <img height="640" alt="Image" src="https://github.com/user-attachments/assets/4a57bf7f-fef3-4580-a0c7-0f3d1e5be851" />

</p>
</details> 

<details><summary>2.02 Screenshots of WAVE tool with Register Page, No Interaction</summary>
<p>

## Register Page, No Interaction
> - Page URL: https://tdm-dev.azurewebsites.net/register
> - Steps to Reproduce: n/a
> - Error: Missing form label

**2.02.01 Screenshot (w/o WAVE Annotations)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/56a989de-69ec-42d7-a75c-50263519f5a8" />

**2.02.02 Screenshot (w/ WAVE Annotations)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/4622dd2e-0594-4879-9b98-775aa19a2d77" />

**2.02.03 Screenshot (click on first Missing form label)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/25887728-4022-4ce9-ad71-f9227d53804c" />

**2.02.04 Screenshot (open Code panel, click on first Missing form label)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/a50670ce-9c9d-4a98-83a8-9ffc29d66c02" />

**2.02.5 HTML DOM Snippet from firstMissing form label**
```
<input name="firstName" type="text" placeholder="First Name" class="form-control " value="">
```
</p>
</details>
